helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -9,37 +9,38 @@ import (
 
 	"github.com/astaxie/beego"
 	"github.com/astaxie/beego/logs"
-
 	"github.com/udistrital/arka_mid/helpers/actaRecibido"
+	"github.com/udistrital/arka_mid/helpers/catalogoElementosHelper"
+	"github.com/udistrital/arka_mid/helpers/cuentasContablesHelper"
 	"github.com/udistrital/arka_mid/helpers/movimientosArkaHelper"
 	"github.com/udistrital/arka_mid/helpers/salidaHelper"
 	"github.com/udistrital/arka_mid/helpers/tercerosHelper"
 	"github.com/udistrital/arka_mid/helpers/tercerosMidHelper"
 	"github.com/udistrital/arka_mid/helpers/ubicacionHelper"
 	"github.com/udistrital/arka_mid/helpers/utilsHelper"
 	"github.com/udistrital/arka_mid/models"
+	"github.com/udistrital/utils_oas/errorctrl"
+	"github.com/udistrital/utils_oas/formatdata"
 	"github.com/udistrital/utils_oas/request"
 )
 
+type InfoCuentasSubgrupos struct {
+	CuentaDebito  *models.CuentaContable
+	CuentaCredito *models.CuentaContable
+}
+
 // RegistrarBaja Crea registro de baja
 func RegistrarBaja(baja *models.TrSoporteMovimiento) (bajaR *models.Movimiento, outputError map[string]interface{}) {
 
-	defer func() {
-		if err := recover(); err != nil {
-			outputError = map[string]interface{}{
-				"funcion": "RegistrarBaja - Unhandled Error!",
-				"err":     err,
-				"status":  "500",
-			}
-			panic(outputError)
-		}
-	}()
+	funcion := "RegistrarBaja"
+	defer errorctrl.ErrorControlFunction(funcion, "500")
 
 	var movimiento *models.Movimiento
 
 	detalleJSON := map[string]interface{}{}
 	if err := json.Unmarshal([]byte(baja.Movimiento.Detalle), &detalleJSON); err != nil {
-		panic(err.Error())
+		funcion += " - json.Unmarshal([]byte(baja.Movimiento.Detalle), &detalleJSON)"
+		return nil, errorctrl.Error(funcion, err, "500")
```

Comment:
(En general) Para preveer posibles logs raros que se podrían registrar como sucesivas concatenaciones de errores, en caso que hayan errores que no sean críticos y se puedan dejar pasar, `funcion` se estaría reescribiendo si se trabaja sobre ella misma...
```suggestion
		eval := " - json.Unmarshal([]byte(baja.Movimiento.Detalle), &detalleJSON)"
		return nil, errorctrl.Error(funcion + eval, err, "500")
```